src/components/ProfileData.js

```diff
@@ -19,12 +19,12 @@ const Container = styled.div`
 
 function ProfileData({ user }) {
   return (
-    <dic>
+    <div>
       <Container>
         <span>Name:</span>
-        <span>Wurstinator</span>
+        <span>{user.name}</span>
       </Container>
-      <Container alt={true}>
+      <Container alt="true">
         <div>
           {user.wins} <hr></hr> Won
```

Comment:
I guess you used <hr> for styling purposes, which I think we shouldn't do. Could try to style the div or a span and then put the user.wins and the Won in there

Reply:
It seperates the number of wins/losses from the respective word. Is that a problem?